### ISSUE TITLE
Tilt during game ending  should not crash 

### ISSUE THREAD

**Issue by jabdoa2** [missionpinball/mpf#1030]
Test tilt during game ending. It crashes during high score.

- Test and probably fix

**Comment by jabdoa2** [comment]
```
context: {'exception': TypeError("'NoneType' object does not support item assignment",), 'message': 'Exception in callback EventManager.process_event_queue()', 'handle': <Handle EventManager.process_event_queue()>}
Traceback (most recent call last):
  File "c:\python34\lib\asyncio\events.py", line 125, in _run
    self._callback(*self._args)
  File "c:\python34\lib\site-packages\mpf\core\events.py", line 615, in process_event_queue
    **event[3])
  File "c:\python34\lib\site-packages\mpf\core\events.py", line 584, in _process_event
    result = self._run_handlers(event, ev_type, kwargs)
  File "c:\python34\lib\site-packages\mpf\core\events.py", line 543, in _run_handlers
    result = handler.callback(**merged_kwargs)
  File "c:\python34\lib\site-packages\mpf\modes\tilt\code\tilt.py", line 134, in reset_warnings
    self.player[self.tilt_config['tilt_warnings_player_var']] = 0
TypeError: 'NoneType' object does not support item assignment
```

**Comment by jabdoa2** [comment]
fixed in 4323891ee5589bc73bc5b4d1617a4991e79b3343
